teacher.rb

```diff
@@ -1,12 +1,12 @@
-require_relative './person'
+require './person'
```

Comment:
- Kindly In order to include classes, modules, etc in other files you have to use require_relative or require, and when it is some dependency on locating in gem kindly use require `ex: require 'date'.` so here above  I suggest you use require_relative instead of require for person `eg: require_relative  './person'`  and  it will make your code looks more professional   :+1: :+1: :+1: